docs/app/services/playground/utilities/system-helper.ts

```diff
@@ -1,104 +1,109 @@
 import { DocumentationType } from '../tokens/documentation.token';
 
+const ANGULAR_VERSION = '9';
+const CDK_VERSION = '9';
+const D3_VERSION = '5.9.2';
+const NGX_BOOTSTRAP_VERSION = '5.3.2';
```

Comment:
Thanks for the changes, they look great, I just noticed an issue regarding ngx-bootstrap.

It looks like we may need to point each component to a separate bundle, eg: for alert it might now need to be https://unpkg.com/browse/ngx-bootstrap@5.5.0/alert/bundles/ngx-bootstrap-alert.umd.js

I haven't tested this so I'm not sure if this is definitely the issue. There as also additional ngx-bootstrap bundles now that we may need to add mappings for, eg: ngx-bootstrap/utils, ngx-bootstrap/locale etc..

As you can see we now get some errors: https://plnkr.co/edit/S4aXywwcctL6THYL

Reply:
Thank you, yes that did seem to resolve the problem.